cmd/server-main.go

```diff
@@ -260,6 +259,8 @@ func isDistributedSetup(eps []*url.URL) bool {
 	return false
 }
 
+const alphabets = "abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
```

Comment:
It is faster and possibly more readable to use a function - 

```go
func IsLetter(string s) bool {
    return len(s) == 1 && ((s[0] >= 'a' && s[0] <= 'z') ||
         (s[0] >= 'A' && s[0] <= 'Z'))
}
```

`strings.Contains()` is significantly more expensive than the above because this is a 52 length constant.